src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -106,20 +124,29 @@ public void Mutate()
 
         public StrykerRunResult Test(StrykerOptions options)
         {
-            var logicalProcessorCount = Environment.ProcessorCount;                  
+            var logicalProcessorCount = Environment.ProcessorCount;
             var usableProcessorCount = Math.Max(logicalProcessorCount / 2, 1);
 
             if (options.MaxConcurrentTestrunners <= logicalProcessorCount)
             {
                 usableProcessorCount = options.MaxConcurrentTestrunners;
             }
 
-            Parallel.ForEach(_input.ProjectInfo.ProjectContents.Mutants.Where(x => x.ResultStatus == MutantStatus.NotRun),
+            var mutantsNotRun = _input.ProjectInfo.ProjectContents.Mutants.Where(x => x.ResultStatus == MutantStatus.NotRun).ToList();
+
+            _progressReporter.ReportInitialState(mutantsNotRun.Count());
+
+            Parallel.ForEach(mutantsNotRun,
                 new ParallelOptions { MaxDegreeOfParallelism = usableProcessorCount },
                 mutant =>
                 {
+                    var timer = new Stopwatch();
```

Comment:
If we won't estimate time left, the custom progress reporter calls can be removed here. Same for the timer.